src/NuGet.Core/NuGet.Packaging/ContentModel/ManagedCodeConventions.cs

```diff
@@ -186,22 +185,24 @@ private static object Locale_Parser(string name, PatternTable table)
                 }
             }
 
+            string frameworkString = name.ToString();
+
             // Check the cache for an exact match
-            if (!string.IsNullOrEmpty(name))
+            if (!string.IsNullOrEmpty(frameworkString))
             {
                 NuGetFramework cachedResult;
-                if (!_frameworkCache.TryGetValue(name, out cachedResult))
+                if (!_frameworkCache.TryGetValue(frameworkString, out cachedResult))
```

Comment:
For future PR, this would be a good table to change to all ReadOnlyMemory<char>, or alternatively, add another cache and only allocate when it's not in the cache.

Reply:
Actually I would absolutely change the key to be `ReadOnlyMemory<char>` in _frameworkCache, and allocate only when the cache lookup misses.

Reply:
Done